test/recipes/81-test_cmp_cli.t

```diff
@@ -24,15 +24,19 @@ BEGIN {
 use lib srctop_dir('Configurations');
 use lib bldtop_dir('.');
 use platform;
+plan skip_all => "These tests are not supported in a fuzz build"
+    if config('options') =~ /-DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION/;
 
```

Comment:
The blank line is on purpose after the first `plan skip_all` because the `use` line before are prerequisites for that  `plan skip_all`.
I've added one more blank like between some `plan skip_all` further below to further indicate their grouping.